source/index.rst

```diff
@@ -23,4 +23,4 @@
    :hidden:
 
    template/index
-
+   install-java
```

Comment:
Two questions:

1. Do we want to move this file to subdirectory `appendix`, then the URL is /appendix/install-java
2. Do we want to rename this file to "java.rst"?

Reply:
My answers are, no for 1 and yes for 2.

Reply:
> 1. Do we want to move this file to subdirectory `appendix`, then the URL is /appendix/install-java

Since we put individual software tutorials in `/source`, it's better to put all those supplementary files in `/source` as well. So `NO`.

> 2. Do we want to rename this file to "java.rst"?

Do we want to rename it as `java/index.rst`?

Reply:
> Do we want to rename it as `java/index.rst`?

OK for me.

Reply:
Okay, I prefer `java/index`. So only `index.rst` is left in `/source`.

Reply:
Done.